benchmarks/ImageProcessing/CMakeLists.txt

```diff
@@ -117,3 +117,29 @@ target_link_libraries(image-processing-benchmark
   # Link Buddy MLIR DIP Library.
   BuddyLibDIP
   )
+
+  #-------------------------------------------------------------------------------
+# Image Processing Resize Benchmark Target
+#-------------------------------------------------------------------------------
+
+add_executable(image-processing-resize-benchmark
```

Comment:
I like this separation of benchmarks for different IP operators. Perhaps we should do that for other operators as well?

Reply:
I think so, im workong on a rotate benchmark. It will be
`image-processing-rotate-benchmark`.